Cartfile

```diff
@@ -1,2 +1,3 @@
 github "SwiftyJSON/SwiftyJSON" ~> 3.1.4
 github "Thomvis/BrightFutures" ~> 5.0
+github "ZeWo/reflection" ~> 0.14
```

Comment:
Quick thought: updating to Swift 3 was a pain, and it seemed like the pain was exponential to the amount of dependencies involved in the project. Is it really worth adding a new dependency here? Is it absolutely required versus implementing this PR's particular reflection needs and wrapping it in an internal class?